conf/defaults.ini

```diff
@@ -825,3 +825,14 @@ use_browser_locale = false
 
 # Default timezone for user preferences. Options are 'browser' for the browser local timezone or a timezone name from IANA Time Zone database, e.g. 'UTC' or 'Europe/Amsterdam' etc.
 default_timezone = browser
+
+
+[sentry]
```

Comment:
I think we should rename this to "frontend logging" or something like that if we replace the sentry lib.
maybe make this a setting under [log] instead?

Reply:
`dsn` is sentry-sepcific, but we can call it `sentry_dsn`

- [ ] change  frontend logging config